test.sh

```diff
@@ -6,12 +6,28 @@
 # TEST_OPENSHIFT_MODE - If set, run OpenShift tests (if present)
 # VERSIONS - Must be set to a list with possible versions (subdirectories)
 
-set -eE
+set -E
```

Comment:
`-E` is not needed if `-e` is not set IIUC.

Reply:
https://github.com/sclorg/container-common-scripts/pull/369/commits/fee1f6a17fc7867f6d7e9fb3d3e36c2cf1340edc